glide.yaml

```diff
@@ -11,11 +11,11 @@ import:
 - package: github.com/giantswarm/operatorkit
   version: master
 - package: github.com/giantswarm/ingresstpr
-  version: master
+  version: structure
```

Comment:
I don't follow why we can't just use master here, add missing Resource methods, and then update ingresstpr. I don't understand how this differs from #20 otherwise.

Reply:
I took the "forget about 19" too literally :smile: now pointing to master, pls take another look